app/Helpers/render/layout.php

```diff
@@ -210,7 +210,7 @@ function RenderToolbar(): void
         echo "<li>";
         echo "<form action='/request/auth/logout.php' method='post'>";
         echo csrf_field();
-        echo "<button class='p-0 bg-transparent text-gray-200 border-0 w-full'>Logout</button>";
+        echo "<button class='btn p-0 bg-transparent text-gray-200 border-0 w-full'>Logout</button>";
```

Comment:
This is in the menu. Do we really want it to look like a button? (I don't actually see any difference)

Reply:
I thought it would require that to make the defaults work. Turned out it'S the other way around because of the global styling from before; it does not need any of those classes anymore because it's fully styled via explicit menu button styles.
Removed in https://github.com/RetroAchievements/RAWeb/pull/1800/commits/debc7dd377d33adc83f24a9c0782c788b265a3a7